assets/js/components/wp-dashboard/WPDashboardWidgets.stories.js

```diff
@@ -125,6 +130,89 @@ ReadyWithCompleteAnalyticsActivationCTA.args = {
 	},
 };
 
+export const ViewOnlyAnalyticsAndSearchConsole = Template.bind( {} );
+ViewOnlyAnalyticsAndSearchConsole.storyName =
+	'View Only Analytics And Search Console';
+ViewOnlyAnalyticsAndSearchConsole.args = {
+	setupRegistry: ( registry ) => {
+		provideModules( registry, [
+			{
+				slug: 'analytics',
+				active: true,
+				connected: true,
```

Comment:
Also in this story file, you can remove `analytics` from here and in 2 other stories bellow

Reply:
Please see the above comments.